model-optimizer/mo/middle/passes/mean_scale_values.py

```diff
@@ -62,20 +62,31 @@ def move_scaleshift_to_preprocess(graph: Graph):
     """
     apply_pattern(
         graph,
-        nodes=[
-            ('weights', dict(kind='data')),
-            ('biases', dict(kind='data')),
+        nodes = [
+            ('input_op', dict(kind='op', op='Parameter')),
             ('input_output', dict(kind='data')),
-            ('scsh_output', dict(kind='data')),
-            ('input_op', dict(kind='op', type='Parameter')),
-            ('scale_shift', dict(kind='op', type='ScaleShift')),
+
+            ('const_mul', dict(kind='op', op='Const')),
+            ('const_mul_output', dict(kind='data')),
+            ('mul', dict(kind='op', op='Mul')),
+            ('mul_output', dict(kind='data')),
+
+            ('const_add', dict(kind='op', op='Const')),
+            ('const_add_output', dict(kind='data')),
+            ('add', dict(kind='op', op='Add')),
+            ('add_output', dict(kind='data')),
         ],
-        edges=[
+        edges = [
             ('input_op', 'input_output'),
-            ('scale_shift', 'scsh_output'),
-            ('input_output', 'scale_shift', {'in': 0}),
-            ('weights', 'scale_shift', {'in': 1}),
-            ('biases', 'scale_shift', {'in': 2}),
+            ('input_output', 'mul', {'in': 0}),
```

Comment:
What if there is just a Mul or just an Add node? I believe that we can create the pre-processing section in this case also.

Reply:
I have not seen any response to this comment. Why it was resolved?

We must support case when we have "input->mul" or "input->add".